e2e/tests/functional/forms.e2e.spec.js

```diff
@@ -91,6 +93,41 @@ test.describe('Persistence operations @addInit', () => {
     });
 });
 
+test.describe('Persistence operations @couchdb', () => {
+    test.use({ failOnConsoleError: false });
+    test('Editing object properties should generate a single persistence operation', async ({ page }) => {
+        test.info().annotations.push({
+            type: 'issue',
+            description: 'https://github.com/nasa/openmct/issues/5616'
+        });
+
+        await page.goto('./', { waitUntil: 'networkidle' });
+
+        const clock = await createDomainObjectWithDefaults(page, {
+            name: `clock-${uuid()}`,
+            type: 'Clock'
+        });
+
+        let putRequestCount = 0;
+        page.on('request', req => {
+            if (req.method() === 'PUT' && req.url().endsWith(clock.uuid)) {
+                putRequestCount += 1;
+            }
+        });
+
```

Comment:
Can add an top level comment as to what we're doing for the next 3 steps

Reply:
Yes! Thanks for the reminder. Let me add some comments.